core/trino-main/src/main/java/io/trino/operator/DriverContext.java

```diff
@@ -275,6 +276,25 @@ public long getPhysicalWrittenDataSize()
         return physicalWrittenBytes;
     }
 
+    public long[] getPartitionPhysicalWrittenDataSize()
+    {
+        // Avoid using stream api for performance reasons
+        int maxSize = 0;
+        List<long[]> partitionPhysicalWrittenDataSizes = new ArrayList<>();
+        for (OperatorContext context : operatorContexts) {
```

Comment:
there should only be one operator in driver that produces `getPartitionPhysicalWrittenDataSize()`,
so appending code seems to be noop